### PR TITLE
Plotly Integration for Bar plot

### PR DESCRIPTION
**Pull request by mail4umar**
- Added code for plotly integration of bar chart
- Added unit tests

### CODE REVIEW

File: verticapy/plotting/_plotly/bar.py [hunk 1]
```diff
@@ -39,18 +41,23 @@ def _compute_method(self) -> Literal["1D"]:
     # Styling Methods.
 
     def _init_style(self) -> None:
-        self.init_style = {}
+        self.init_trace_style = {"marker_color": PlotlyBase.get_colors(idx=0)}
```
Comment (oualib) [inline verticapy/plotting/_plotly/bar.py:44]: your class is inheriting from PlotlyBase, you should write self.get_colors

File: verticapy/plotting/_plotly/bar.py [hunk 2]
```diff
@@ -39,18 +41,23 @@ def _compute_method(self) -> Literal["1D"]:
...
+        self.init_layout_style = {"width": 200*len(self.layout['labels']), "height": 500}
```
Comment (oualib) [inline verticapy/plotting/_plotly/bar.py:45]: use 'black' to reindent your code.

File: verticapy/tests/vDataFrame/test_vDF_plot_plotly.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        result=titanic_vd["survived"].bar()
+        assert(type(test_fig)==plotly.graph_objs._figure.Figure)
+        assert(type(result)==plotly.graph_objs._figure.Figure)
+        assert(test_fig.data[0]['y'][0]/test_fig.data[0]['y'][1]==result.data[0]['y'][0]/result.data[0]['y'][1])
+        assert(test_fig.data[0]['x'][0]==result.data[0]['x'][0])
+        assert(test_fig.layout['xaxis']['type']=='category')
+        result=titanic_vd["survived"].bar(xaxis_title="Custom X Axis Title")
+        assert(result.layout['xaxis']['title']['text']=='Custom X Axis Title')
+        result=titanic_vd["survived"].bar(yaxis_title="Custom Y Axis Title")
+        assert(result.layout['yaxis']['title']['text']=='Custom Y Axis Title')
```
Comment (oualib) [inline verticapy/tests/vDataFrame/test_vDF_plot_plotly.py:76]: In the future, we should build more robust & comprehensible unit tests. Do not hesitate to write comments.

File: verticapy/tests/vDataFrame/test_vDF_plot_plotly.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+import datetime, os, sys
+
+# Other Modules
+import plotly.express as px
+import plotly
+
+# VerticaPy
+import verticapy
+import verticapy._config.config as conf
+from verticapy import drop, set_option
```
Comment (oualib) [inline verticapy/tests/vDataFrame/test_vDF_plot_plotly.py:31]: why loading so many things if you are just using few. Import only what is needed.

**Review by oualib — CHANGES_REQUESTED** [review]

File: verticapy/plotting/_plotly/bar.py (hunk 1 above)
Comment (mail4umar) [reply verticapy/plotting/_plotly/bar.py:44]: done

File: verticapy/tests/vDataFrame/test_vDF_plot_plotly.py (hunk 4 above)
Comment (mail4umar) [reply verticapy/tests/vDataFrame/test_vDF_plot_plotly.py:31]: done

Comment (oualib) [reply verticapy/tests/vDataFrame/test_vDF_plot_plotly.py:30]: you're importing config as conf. Use:
 - conf.set_option
No need to import set_option.

**Review by oualib — CHANGES_REQUESTED** [review]
Please address my two last comments.

**Review by oualib — APPROVED** [review]

**Pull request merged**